packages/docs/.vuepress/config.js

```diff
@@ -0,0 +1,79 @@
+// .vuepress/config.js
+module.exports = {
+  theme: '@okta/vuepress-theme-nimatron',
+  dest: 'dist',
+  head: [
+    [
+      'meta', { 
+        'og:title': 'Odyssey, the Okta design system',
+        'og:type': 'article',
+        'og:url': 'https://design-docs.trexcloud.com',
+        'og:image': 'https://www.okta.com/sites/default/files/okta-logo-onblue.png',
+      },
+    ],
+    [
+      'script', { 
+        'src': 'https://unpkg.com/micromodal@0.3.2/dist/micromodal.min.js',
+      }
+    ],
+    [
+      'script', { 
+        'src': 'https://unpkg.com/choices.js@9.0.1/public/assets/scripts/choices.min.js',
```

Comment:
Since this is a public `npm` package - is there a reason we cannot reference this dependency through our local `node_modules`?

Reply:
@jmelberg-okta i just realized choices is MIT; does that make it a problem for direct inclusion?

Reply:
There shouldn't be any problems for using it - this comment is more of suggesting how we consume it.

I'd expect that VuePress allows us to reference this from our `node_modules` directory instead of directly from a public URL via a network request. If we embed it as a dependency for our docs package, we should be able to avoid that network call.

Reply:
Yep this is a fair call out, I have put them them in to their respective vuepress plugins.

This is fixed.